designer/server/src/main/scala/pl/touk/nussknacker/ui/server/AkkaHttpBasedRouteProvider.scala

```diff
@@ -401,7 +401,7 @@ class AkkaHttpBasedRouteProvider(
 
       val usageStatisticsReportsConfig = resolvedConfig.as[UsageStatisticsReportsConfig]("usageStatisticsReports")
       val fingerprintService           = new FingerprintService(new FingerprintRepositoryImpl(dbRef))
-      val usageStatisticsReportsSettingsDeterminer = UsageStatisticsReportsSettingsDeterminer(
+      val usageStatisticsReportsSettingsDeterminer = UsageStatisticsReportsSettingsService(
```

Comment:
Should we also change the name of variable?

Reply:
Done